ModelBundle/DataFixtures/MongoDB/LoadContentData.php

```diff
@@ -134,7 +170,8 @@ public function generateCustomerConvenant($language)
         $content->setStatus($this->getReference('status-published'));
         $content->setVersion(2);
         $content->addKeyword(EmbedKeyword::createFromKeyword($this->getReference('keyword-lorem')));
-        $content->addKeyword(EmbedKeyword::createFromKeyword($this->getReference('keyword-sit')));
```

Comment:
Why delete this keyword ? It is required to get various and coherent functionnal testing :s